### ISSUE TITLE
Difference in edit user account pages.

### ISSUE THREAD

**Issue by duncangraham** [hivemindworks/Argo-Project-V0#117]
Hey @freindst can you explain to me the difference between editing my information on the user/edit endpoint vs the payment/updateClientToken endpoint?

its the same information (name and phone number), so why are there 2 places to update this?

## user/edit
<img width="1391" alt="screen shot 2015-08-13 at 1 46 15 pm" src="https://cloud.githubusercontent.com/assets/865298/9261306/ca6ae870-41c1-11e5-999a-58f6d445697e.png">

## payment/updateClientToken
<img width="1391" alt="screen shot 2015-08-13 at 1 46 31 pm" src="https://cloud.githubusercontent.com/assets/865298/9261312/dab65318-41c1-11e5-8dbc-a74bc2d2c702.png">

**Comment by markstm17** [comment]
@freindst and @duncangraham 

From my experience becoming the first Argo user (with all updates), I personally went out of my way to update my personal information. I put in my name and cell number, but the vast majority/most people won't do that, they will do that as they are pledging for a trip.

When I then pledged for the trip I just created it didn't remember my name and number from when I inputted them moments earlier. For the time being, I don't think we should have two places to edit your info. Let the pledges fill in their names and number and let that automatically update to their account page, but most people won't be getting new numbers and even fewer people will be changing their names, so I think we should freeze the ability for people to change or edit this information. 

The first bus they too should be when that individual is required to put in First and Last name and Number, then CC info
